### PR TITLE
Updates To BCMP CLI For Grammar/Spelling Fixes And Optimization

### PR DESCRIPTION
**Pull request by matt001k**
Reduces code complexity in set message
Fixes spelling on commit message print statement

### CODE REVIEW

File: src/lib/bm_integration/bcmp_cli.cpp [hunk 1]
```diff
@@ -224,74 +224,46 @@ static BaseType_t cmd_bcmp_fn(char *writeBuffer, size_t writeBufferLen,
...
-          }
+          float val = strtof(const_cast<char *>(value_str), NULL);
           if (cbor_encode_float(&encoder, val) != CborNoError) {
-            break;
-          }
-          if (!bcmp_config_set(node_id, partition, key_str_str_len, key_str, buffer_size,
-                               cbor_buf, &err, NULL)) {
-            printf("Failed to send message config get\n");
-          } else {
-            printf("Succesfully sent config get msg\n");
```
Comment (matt001k) [inline src/lib/bm_integration/bcmp_cli.cpp:263]: This statement was also incorrect as this is for the `set` message.

File: src/lib/bm_integration/bcmp_cli.cpp [hunk 2]
```diff
@@ -318,7 +290,7 @@ static BaseType_t cmd_bcmp_fn(char *writeBuffer, size_t writeBufferLen,
         if (!bcmp_config_commit(node_id, partition, &err)) {
           printf("Failed to send config commit\n");
         } else {
-          printf("Succesfull config commit send\n");
+          printf("Succesfully config commit send\n");
```
Comment (towynlin) [inline src/lib/bm_integration/bcmp_cli.cpp:293]:
```suggestion
          printf("Succesfully sent config commit\n");
```

File: src/lib/bm_integration/bcmp_cli.cpp [hunk 3]
```diff
@@ -224,74 +224,46 @@ static BaseType_t cmd_bcmp_fn(char *writeBuffer, size_t writeBufferLen,
...
           break;
         }
         default:
           break;
         }
+        if (!bcmp_config_set(node_id, partition, key_str_str_len, key_str, buffer_size,
+                             cbor_buf, &err, NULL)) {
+          printf("Failed to send message config set\n");
+        } else {
+          printf("Succesfully sent config set msg\n");
```
Comment (towynlin) [inline src/lib/bm_integration/bcmp_cli.cpp:265]: It's all fine, but here are a couple nit picks:

- If we need both cases anyway, then the `!` in the if expression just seems confusing to me. I'd prefer to remove it and swap the success and failure clauses.
- Let's use the same phrasing in both the success and failure cases, for example, "Failed to send config set message" and "Successfully sent config set message".

**Review by towynlin — APPROVED** [review]
Looks good. Had a couple nits. Make sure to manage the submodule carefully — merge bm_core first, then update submodule here to the merge commit, then merge this PR.

**Review by victorsowa12 — APPROVED** [review]

File: src/lib/bm_integration/bcmp_cli.cpp (hunk 3 above)
Comment (matt001k) [reply src/lib/bm_integration/bcmp_cli.cpp:265]: Agreed, that sounds good!

**Pull request merged**